src/components/calcite-action-pad/calcite-action-pad.tsx

```diff
@@ -51,15 +59,29 @@ export class CalciteActionPad {
   // --------------------------------------------------------------------------
 
   render() {
-    const { offsetTop } = this;
+    const { el, offsetTop } = this;
 
     const style = {
       top: `${offsetTop}px`
     };
 
+    const rtl = getElementDir(el) === "rtl";
+
+    const closest = el.closest("calcite-shell-panel");
+    const layout = (closest && closest.layout) || "leading";
+
     return (
-      <Host style={style}>
-        <slot />
+      <Host>
+        <div
```

Comment:
Added a div here because I don't think we should be adding classes or styles to the host element, right? May need updating in other places

Reply:
checked and other places look good

Reply:
👍